src/storage/storage.hpp

```diff
@@ -178,13 +227,41 @@ class Storage : public aos::sm::launcher::StorageItf,
         }
     };
 
+    struct CertInfo {
+        uint8_t  mIssuer[aos::crypto::cCertIssuerSize];
+        size_t   mIssuerLen;
```

Comment:
mIssuerSize
mSerialSize

We use size for bytes and len for string.

Reply:
Done